datafusion/physical-expr/src/expressions/cast.rs

```diff
@@ -740,6 +740,101 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    fn test_cast_timestamp_with_timezone_to_timestamp() -> Result<()> {
```

Comment:
These ended up not being necessary (the bug was elsewhere in my changes) but I think it's good coverage anyway so leaving it in here.

Reply:
These seem like they could be just as easily tested with `.slt` / `arrow_cast` calls 🤔 

 That would be less verbose and easier to maintain in my opinion